src/app/app.component.js

```diff
@@ -30,15 +31,35 @@ function AppComponentCtrl(
 
     this.settings = dimSettingsService;
     $scope.$watch(() => this.settings.itemSize, (size) => {
-      document.querySelector('html').style.setProperty("--item-size", `${size}px`);
+      if (!isPhonePortrait()) {
+        document.querySelector('html').style.setProperty("--item-size", `${size}px`);
+      }
```

Comment:
In each of these, you probably need to clear the property if it *is* `isPhonePortrait`.